tests/test_command_copilot.py

```diff
@@ -418,3 +415,12 @@ def _put_ssm_param(client, app, env, name, value):
     assert SSM_PATH.format(app="myapp", env="anotherenv", name="OTHER_ENV") not in result.output
     validate_version.assert_called_once()
     assert result.exit_code == 0
+
+
+def create_test_manifests(addon_file_contents, fakefs):
```

Comment:
This could live in a utils file

Reply:
I think this method is actually fairly specific to the running of the `make-addons` tests. I'm not sure moving it would be right.